src/cli/args.js

```diff
@@ -1,5 +1,15 @@
-const parseArgs = () => {
-    // Write your code here 
+import * as readline from 'readline/promises'
+
+const parseArgs = async() => {
+    const readlineInterface = readline.createInterface({input: process.stdin, output: process.stdout})
+    const answer = await readlineInterface.question('Print arguments in format (--propName value --prop2Name value2):')
+
+    const regular = /--(\w+)\s(\w+)/g
```

Comment:
this regular expression doesn't work with the example from package.json "--some-arg value1"

Reply:
there is no such case in the task description. All parameters with camelCase as this prop2Name